### ISSUE TITLE
EmojiList should be keyboard accessible

### ISSUE THREAD

**Issue by LukeGarrigan** [jm-david/emoji-mart-vue#79]
When focusing on the emoji-picker the user should be able to use arrow-keys or tab to navigate through the list of emojis. 

A similar change was made to the main parent of this project (emoji-mart)
Issue: https://github.com/missive/emoji-mart/issues/218
PR: https://github.com/missive/emoji-mart/pull/284 

It would be great if a similar change would be made in this project!
